definitions/ext-sap_mssql/golden_metrics.yml

```
@@ -0,0 +1,33 @@
+d0bAvailability:
```

```suggestion
dbAvailability:
```

Also since these are already DB's shouldn't this be availability only?

---

Yes. I made the change to took out the "db" in the titles and section label.